doc/distrib/NodeHelpFiles/_review/Revit.Elements.TextNote.Height.md

```diff
@@ -0,0 +1,9 @@
+## In Depth
+`TextNote.Height` returns the given text note's height in project units.
+
+In the example below, all working notes are collected from the current Revit document. The height values are then returned.
```

Comment:
What is the unit of these values?

Reply:
Added more info

Reply:
`TextNote.Height` returns the given text note's height in project units.

In the example below, all working notes are collected from the current Revit document. The height values are then returned in project units _(feet and inches)_.